static/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -462,7 +460,23 @@ function ConfigureIntegration({params, router, routes, location}: Props) {
         ]
       : [];
 
-    const allTabs = tabs.concat(codeOwnerTabs);
+    // User mappings are mutually exclusive with stacktrace linking
+    // and code owners, so only render the main settings tab and user mappings.
+    const userMappingTabs: Array<[Tab, string]> = hasUserMapping
+      ? [
+          ['repos', t('Settings')],
```

Comment:
This is a little quirky, but the main "settings" tab is called `repos` in the UI